randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -103,6 +105,97 @@ def get_resources_for_details(
     return resources
 
 
+def _construct_music_shuffle_dict(music_mode: MusicMode, rng: Random) -> dict[str, str]:
+    combat_list = [
+        "boss_go",
+        "boss_mae_32",
+        "m_boss_alpha_mn",
+        "m_boss_alpha2_99",
+        "m_boss_arachnus03",
+        "m_boss_gamma04",
+        "m_boss_manic_miner_bot99",
+        "m_boss_manic_miner_chase99",
+        "m_boss_omega02",
+        "m_boss_queen02",
+        "m_boss_ridley_second99",
+        "m_boss_ridley_third99",
+        "m_boss_ridley01",
+        "m_boss_zeta06",
+    ]
+
+    exploration_list = [
+        "ele",
+        "k_brinstra_0714",
+        "k_crateria99",
+        "k_matad_result",
+        "k_ravacaves2_0916",
+        "m_allarea_a99",
+        "m_allarea_b99",
+        "m_area_indefinite01_99",
+        "m_area_tension02",
+        "m_area1_add1_03",
+        "m_area2_add1_02",
+        "m_area2_basic02",
+        "m_area4_5_basic01",
+        "m_area6_basic01",
+        "m_area7_8_basic01",
+        "m_area10_basic02",
+        "m_boss_before99",
+        "m_chozo_gallery_las",
+        "m_met2_ancient_chozo_ruins_2_99",
+        "m_met2_ancient_chozo_ruins99",
+        "m_met2_caverns1_99",
+        "m_met2_caverns3_99",
+        "m_met2_caverns4_99",
+        "m_met2_metroid_hatching99",
+        "m_prologue99",
+        "m_select99",
+        "m_title99",
+        "t_m2_ancient_chozo_ruins_arr2_lp",
+        "t_m2_surface_arr1",
+        "t_matad_ambi06",
+        "t_matad_area01_damage",
+    ]
+
+    fanfare_list = [
+        "event_jingle_placeholder",
+        "k_matad_jinchozo",
+        "matad_jintojo_32728k",
+        "special_ability2_32",
+        "sphere_jingle_placeholder",
+        "tank_jingle",
+    ]
```

Comment:
Like already discussed, please shuffle fanfares even in full mode only with one of the same list.